server/etcdserver/apply.go

```diff
@@ -428,13 +428,18 @@ func (a *applierV3backend) Range(ctx context.Context, txn mvcc.TxnRead, r *pb.Ra
 }
 
 func (a *applierV3backend) Txn(ctx context.Context, rt *pb.TxnRequest) (*pb.TxnResponse, *traceutil.Trace, error) {
+	var txn mvcc.TxnWrite
```

Comment:
Can you move this right above the line 438? It's not used in the tracing block.

Reply:
> Can you move this right above the line 438? It's not used in the tracing block.

Done.